### DISCUSSION TITLE
Is it possibility to run harbor in single container or bare machine ?

### DISCUSSION THREAD

**Discussion by Alceatraz** [goharbor/harbor#17103] [Ideas]
For example, Jenkins Gitlab can deploy in single docker container. Or they can deploy into the system directly. Is there any way deploy harbor like that?

**Comment by Vad1mo** [comment]
You can, however, you are on your own here. There is no documentation or resources to do so. Reason is that most don't do it that way, or they just use docker distribution instead of harbor.

**Reply by Alceatraz** [reply]
Actually I want find something can host helm charts, Harbor need to many components, Nexus can't upload by cli buy only webui manual upload, And chart museum need plugin, which make helm repo almost another type of registry. As far as I know only harbor support native helm push
